### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures aria-hidden elements do not contain focusable elements (#slick-slide03)

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#7]
#### Issue
`Ensures aria-hidden elements do not contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[Personal Banking & Netbanking Services Online - ICICI Bank](https://www.icicibank.com/?ITM=nli_cms_HP_logo_personal_topnavigation)
#### Element path
#slick-slide03
#### Snippet
`<div class="banner-list slick-slide" data-slick-index="3" aria-hidden="true" tabindex="0" role="tabpanel" id="slick-slide03" aria-describedby="slick-slide-control03" style="width: 2048px;" data-gtm-vis-first-on-screen-32405525_83="7930" data-gtm-vis-fir...`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 100.0.4896.127
This accessibility issue was found using Accessibility Insights for Web 2.31.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
